lua/tardis/languages/en.lua

```diff
@@ -93,6 +93,7 @@ T.Phrases = {
     ["Common.Save"] = "Save",
     ["Common.Set"] = "Set",
     ["Common.TARDIS"] = "TARDIS",
+    ["Common.ARTRON"] = "ARTRON",
```

Comment:
This should be `Common.Atron`, `TARDIS` is only uppercase above as it is an acronym

Reply:
Fixed